Packs/Code42/Playbooks/playbook-Code42_Change_Blocked_Status_README.md

```diff
@@ -0,0 +1,33 @@
+A simple playbook for blocking or unblocking a user in Code42.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* Code42
+
+### Scripts
+This playbook does not use any scripts.
+
+### Commands
+* code42-user-unblock
+* code42-user-block
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Action | Either the string "block" or "unblock". |  | Optional |
+| username | The username of the user to block or unblock in Code42. |  | Optional |
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Code42 Change Blocked Status](Insert the link to your image here)
```

Comment:
todo

Reply:
What is weird is that the original Playbooks were this way.